src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -708,19 +704,19 @@ template <typename EventType, typename StructType>
 
 AmdgpuCsIoctlPerfEvent ConsumeAmdgpuCsIoctlPerfEvent(PerfEventRingBuffer* ring_buffer,
                                                      const perf_event_header& header) {
-  return ConsumeGpuEvent<AmdgpuCsIoctlPerfEvent, amdgpu_cs_ioctl_tracepoint>(ring_buffer, header);
+  return ConsumeGpuEvent<AmdgpuCsIoctlPerfEvent, AmdgpuCsIoctlTracepoint>(ring_buffer, header);
```

Comment:
This shows you what I mean: now it's way less clear that one is a `PerfEvent` and the other one is the raw memory.

Reply:
Discussed offline